app/src/main/kotlin/com/wire/android/ui/theme/WireDimensions.kt

```diff
@@ -48,7 +77,27 @@ private val DefaultPhonePortraitWireDimensions: WireDimensions = WireDimensions(
     buttonMinHeight = 48.dp,
     buttonMinWidth = 60.dp,
     buttonHorizontalContentPadding = 16.dp,
-    buttonVerticalContentPadding = 8.dp
+    buttonVerticalContentPadding = 8.dp,
+    userProfileLogoutBtnHeight = 32.dp,
+    userProfileStatusBtnHeight = 32.dp,
+    userProfileOtherAccItemHeight = 56.dp,
+    spacing4 = 4.dp,
+    spacing8 = 8.dp,
+    spacing16 = 16.dp,
+    spacing24 = 24.dp,
+    spacing32 = 32.dp,
+    spacing40 = 40.dp,
+    spacing48 = 48.dp,
+    spacing56 = 56.dp,
+    spacing64 = 64.dp,
+    corner2 = 2.dp,
+    corner4 = 4.dp,
+    corner6 = 6.dp,
+    corner8 = 8.dp,
+    corner10 = 10.dp,
+    corner12 = 12.dp,
+    corner14 = 14.dp,
+    corner16 = 16.dp
```

Comment:
I'm still not sure which way should we use, both have its pros and cons - if we create separate dimension for every ui item then we can modify them independently but we will end up with having a lot of dependencies, with this approach we have less dimensions but for instance if we would want to support tablet screen sizes, and we use this dimension everywhere in the app then it will change also in places that we would not want it to be changed
Maybe for now we can use it the way you implemented it but I would like to know the opinion of the rest of the team 😄

Reply:
I think we should extract hardcoded dimensions from elements that are either reusable by other composable objects, or elements whose size is meant to be changed when implementing for other devices, sizes (internal paddings, width and height. 
Therefore I'm not really sure if having a semantic approach (corner12 -> 12 DP's) when extracting these dimensions makes real sense as it will not be very logic to assign `corner12 = 24.dp` on a tablet. If so, I would maybe go for a `cornerRadius1x`, `cornerRadius2x`... approach.

Reply:
@gongracr so, for example, we went to `cornerRadius1x` solution...
So it's gonna be like:            
```
for phone                     and for Tablets:
cornerRadius1x = 2.dp                cornerRadius1x = 4.dp
cornerRadius2x = 4.dp                cornerRadius2x = 8.dp
cornerRadius3x = 6.dp                cornerRadius3x = 12.dp
...
```

So what is the big difference from this 
```
corner2 = 2.dp               corner2 = 4.dp
corner4 = 4.dp               corner4 = 8.dp
corner6 = 6.dp               corner6 = 12.dp
...
```

Reply:
That for example the corner6 might give the false assumption that it is always 6.dp. Again this responds to more a personal preference than to a well valid argument. I will not oppose if you prefer leaving it like that.